apis/apps/v1alpha1/nebulabackup_types.go

```diff
@@ -98,6 +98,12 @@ type BackupSpec struct {
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
 	BR *BRConfig `json:"br,omitempty"`
+
+	// MaxBackups is to specify how many backups we want to keep. Used for scheduled backups only.
+	MaxBackups *int32 `json:"maxBackups,omitempty"`
+
+	// MaxReservedTime is to specify how long backups we want to keep. Used for scheduled backups only.
+	MaxReservedTime *string `json:"maxReservedTime,omitempty"`
```

Comment:
redundant fields

Reply:
@MegaByte875 These are needed to pass the MaxBackups and MaxReservedTime to NebulaBackup so it can pass them to the script. Will change the variable names and make this more clear.

Reply:
Fixed